sdv/metadata/single_table.py

```diff
@@ -300,6 +300,28 @@ def update_column(self, column_name, **kwargs):
         self.columns[column_name] = _kwargs
         self._updated = True
 
+    def update_columns(self, column_names, **kwargs):
+        """Update multiple columns with the same metadata kwargs.
+
+        Args:
+            column_names (list[str]):
+                A list of column names to be updated.
+            **kwargs (type):
+                Any key word arguments that describe metadata for the column.
+        """
+        for column_name in column_names:
+            self.update_column(column_name, **kwargs)
+
+    def update_columns_metadata(self, column_metadata):
+        """Update the metadata for multiple columns using metadata from the input dictionary..
```

Comment:
minor: typo, remove extra .

Reply:
Yes, done in [f2a4dc4](https://github.com/sdv-dev/SDV/pull/1824/commits/f2a4dc4a6a57c0349921d4647e663638b16e944b)